src/components/course/CourseRunCard.jsx

```diff
@@ -171,12 +171,15 @@ const CourseRunCard = ({
       : 'Be the first to enroll!';
 
     let tempHeading = `${isCourseStarted ? 'Started' : 'Starts'} ${moment(start).format(DATE_FORMAT)}`;
+
     if (isCourseSelfPaced(pacingType)) {
-      if (isCourseStarted && hasTimeToComplete(courseRun)) {
+      if (isCourseStarted) {
```

Comment:
Nit: I don't think you need to separate these conditionals in to different statements?